doc/development/01docker+local.md

```diff
@@ -0,0 +1,25 @@
+# Docker + local
+
+Run the database in Docker and the frontend and backend locally.
+
+## Install Docker
+
+[Install Docker](https://docs.docker.com/engine/install/) for your host OS.
+
+## Installing Node 20 + Npm
+
+Make sure node 20 is installed, you may use to easly install multiple versions of node [nvm](https://github.com/nvm-sh/nvm),
```

Comment:
Try to not have several mentions of which versions to install, this is hard to maintain. (Add link to where the version is already mentioned).

```suggestion
Make sure node, in the version as required for PermaplanT, is installed, you may use to easily install multiple versions of node [nvm](https://github.com/nvm-sh/nvm),
```